github/pkg/apis/sources/v1alpha1/githubsource_types.go

```diff
@@ -130,6 +130,10 @@ const (
 	// GitHubSource has been configured with event types.
 	GitHubSourceConditionEventTypesProvided apis.ConditionType = "EventTypeProvided"
 
+	//GitHubSourceConditionWebhookProvided has a status True when the
+	//GitHubSource has been configured with a webhook.
+	GitHubSourceConditionWebhookProvided apis.ConditionType = "WebhookProvided"
```

Comment:
I think I'd use `Configured` vs. `Provided` everywhere.  `Provided` makes it seem like it's on the user to _provide_ it, and I could see that leading to confusion.